docs/content/preview/yugabyte-platform/alerts-monitoring/universe-logging.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Export PostgreSQL logs from YugabyteDB Anywhere universes
+headerTitle: Export audit logs
+linkTitle: Export logs
+description: Export universe logs to third-party tools.
+headcontent: Export PostgreSQL logs from your universe
+badges: ea
+menu:
+  preview_yugabyte-platform:
+    identifier: universe-logging
+    parent: alerts-monitoring
+    weight: 70
+type: docs
+---
+
+Export YSQL database logs to third-party tools for security monitoring, to build operations and health dashboards, troubleshooting, and more. You can export the following types of logs:
+
+<!-- Database query logging. This is the standard [PostgreSQL logging](https://www.postgresql.org/docs/11/runtime-config-logging.html) facility. Using these settings you can log query statements and errors.-->
+- Database audit logging. Using the [PostgreSQL Audit Extension](https://www.pgaudit.org/#) ([pgaudit](https://github.com/pgaudit/pgaudit/blob/1.3.2/README.md)), the audit log provides the exact database transactions, which is a compliance requirement for government, financial, or ISO certifications.
+
+Note that YugabyteDB is based on PostgreSQL 11 and uses pgaudit v1.3.2.
+
+## Prerequisites
+
+The Audit log export feature is {{<badge/ea>}}. To enable the feature in YugabyteDB Anywhere, set the **Enable DB Audit Logging** Global Configuration option (config key `yb.universe.audit_logging_enabled`) to true. Refer to [Manage runtime configuration settings](../../../administer-yugabyte-platform/manage-runtime-config/). Note that only a Super Admin user can modify Global configuration settings. The flag can't be turned off if audit logging is enabled on a universe.
+
+To export logs, you need to first create an export configuration. A configuration defines the sign in credentials and settings for the tool that you want to export your logs to. Refer to [Manage export configurations](../anywhere-export-configuration).
+
+## Limitations
+
+- Log export is only supported for cloud-based universes and automatically provisioned on-premises universes. Manually provisioned universes and Kubernetes universes are not supported.
+- Log export uses a PostgreSQL extension and is only compatible with YSQL.
```

Comment:
rephrase second limitation to, "Only YSQL database audit logs can be exported, not YCQL"